packages/component/src/SendBox/SuggestedActions.js

```diff
@@ -46,10 +58,7 @@ export default connectSuggestedActions(
         ) =>
           <SuggestedAction
             buttonText={
-              type === 'messageBack' ?
-                title || displayText
-              :
-                title || (typeof value !== 'string') ? JSON.stringify(value) : value
+             suggestedActionText({displayText, title, type, value})
```

Comment:
Formatting: parenthesis between bracelets.

```suggestion
             suggestedActionText({ displayText, title, type, value })
```